gapii/cc/pack_encoder.cpp

```diff
@@ -30,70 +31,77 @@ using ::google::protobuf::Message;
 using ::google::protobuf::Descriptor;
 using ::google::protobuf::DescriptorProto;
 using ::google::protobuf::io::CodedOutputStream;
-using ::google::protobuf::io::StringOutputStream;
 
 namespace {
 
-const int      VERSION_MAJOR       = 1;
-const int      VERSION_MINOR       = 1;
-const int64_t  TAG_FIRST_GROUP     = -1;
-const int64_t  TAG_GROUP_FINALIZER = 0;
-const int64_t  TAG_DECLARE_TYPE    = 1;
-const int64_t  TAG_FIRST_OBJECT    = 2;
-const uint64_t NO_ID               = 0xffffffffffffffff;
+const int      VERSION_MAJOR          = 1;
+const int      VERSION_MINOR          = 1;
+const int64_t  TAG_FIRST_GROUP        = -1;
+const int64_t  TAG_GROUP_FINALIZER    = 0;
+const int64_t  TAG_DECLARE_TYPE       = 1;
+const int64_t  TAG_FIRST_OBJECT       = 2;
+const uint64_t NO_ID                  = 0xffffffffffffffff;
```

Comment:
Übernit: Undo pointless whitespace changes for git blame's sake.

Reply:
I was re-indenting for some constants that I since removed, left the indent in by mistake.
Moving forward, when adding a new constant that goes past the upper bound of indentation for previous constants, should all constants be re-indented?